languages/default.pot

```diff
@@ -1,42 +1,34 @@
-#, fuzzy
+# Copyright (C) 2022 
+# This file is distributed under the same license as the Disciple.Tools - Prayer Campaigns plugin.
 msgid ""
 msgstr ""
-"Project-Id-Version: Disciple Tools - Subscriptions\n"
-"POT-Creation-Date: 2022-09-20 11:49+0100\n"
-"PO-Revision-Date: 2018-01-29 05:53-0700\n"
-"Last-Translator: \n"
-"Language-Team: \n"
+"Project-Id-Version: Disciple.Tools - Prayer Campaigns 2.0.2\n"
+"Report-Msgid-Bugs-To: https://wordpress.org/support/plugin/disciple-tools-prayer-campaigns\n"
+"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
+"Language-Team: LANGUAGE <LL@li.org>\n"
 "MIME-Version: 1.0\n"
 "Content-Type: text/plain; charset=UTF-8\n"
 "Content-Transfer-Encoding: 8bit\n"
-"Plural-Forms: nplurals=INTEGER; plural=EXPRESSION;\n"
-"X-Generator: Poedit 3.1.1\n"
-"X-Poedit-Basepath: ..\n"
-"X-Poedit-Flags-xgettext: --add-comments=translators:\n"
-"X-Poedit-WPHeader: disciple-tools-prayer-campaigns.php\n"
-"X-Poedit-SourceCharset: UTF-8\n"
-"X-Poedit-KeywordsList: ;__;_e;_n:1,2;_x:1,2c;_ex:1,2c;_nx:4c,1,2;"
-"esc_attr__;esc_attr_e;esc_attr_x:1,2c;esc_html__;esc_html_e;"
-"esc_html_x:1,2c;_n_noop:1,2;_nx_noop:3c,1,2;__ngettext_noop:1,2\n"
-"X-Poedit-SearchPath-0: .\n"
-"X-Poedit-SearchPathExcluded-0: *.js\n"
-"X-Poedit-SearchPathExcluded-1: */libraries/*\n"
-"X-Poedit-SearchPathExcluded-2: disciple-tools-prayer-campaigns.php\n"
-"X-Poedit-SearchPathExcluded-3: post-type/list-page-tile.php\n"
-"X-Poedit-SearchPathExcluded-4: post-type/module-subscriptions-management."
-"php\n"
-"X-Poedit-SearchPathExcluded-5: post-type/loader.php\n"
-"X-Poedit-SearchPathExcluded-6: post-type/dt-subscriptions.php\n"
-"X-Poedit-SearchPathExcluded-7: post-type/module-subscriptions-base.php\n"
-"X-Poedit-SearchPathExcluded-8: post-type/campaign_core.js\n"
-"X-Poedit-SearchPathExcluded-9: post-type/module-campaigns-24hour-prayer."
-"php\n"
-"X-Poedit-SearchPathExcluded-10: admin\n"
-"X-Poedit-SearchPathExcluded-11: classes\n"
-"X-Poedit-SearchPathExcluded-12: post-type\n"
```

Comment:
@squigglybob i did have a few files excluded

Reply:
and folders

Reply:
excluded those three